_test/tests/inc/sort_without_collator.test.php

```diff
@@ -2,161 +2,85 @@
 
 use dokuwiki\Utf8\Sort;
 
+require_once __DIR__ . '/sort_with_collator.test.php';
```

Comment:
INFO: as a former Java programmer, this `require_once __DIR__` looks awful to me :-)

Reply:
yeah, at some point we'll have to refactor the test suit to use proper autoloading, but there are more pressing things